src/lib.rs

```diff
@@ -1310,7 +1329,7 @@ impl Drop for Registration {
 impl Configuration {
     pub fn new(
         registration: &Registration,
-        alpn: &Buffer,
+        alpn: &[Buffer],
```

Comment:
Why the changes in this section?

Reply:
Configuration::new Rust function was missing a way to actually give more than one alpn parameter, so it had to be changed anyways, so I'm proposing here using Rust slice rather than reference and separate length. Listener `start` is another place alpn is passed so I changed that as well to be uniform.

Reply:
Ok.